docs/template_plugin/src/CMakeLists.txt

```diff
@@ -23,11 +23,18 @@ target_include_directories(${TARGET_NAME} PRIVATE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${IE_MAIN_TEMPLATE_PLUGIN_SOURCE_DIR}/include")
 
-target_link_libraries(${TARGET_NAME} PRIVATE IE::inference_engine IE::inference_engine_transformations ${NGRAPH_LIBRARIES} ${INTEL_ITT_LIBS})
+target_link_libraries(${TARGET_NAME} PRIVATE
+    IE::inference_engine
+    IE::inference_engine_transformations
+    ${INTEL_ITT_LIBS}
+    ${NGRAPH_LIBRARIES}
+    ngraph_backend
+    interpreter_backend
```

Comment:
Do we really want to link template plugin with some libraries for tests? Maybe should we add comments in order to warn user that it is not required component and he shouldn't use it in his own plugin.

Reply:
BTW `interpreter_backend` supports only limited scope of operations.

Reply:
Moved to separate section. Should not `interpreter_backend` supports all operations from current opset as reference code as it is used for tests?

Reply:
At the current moment Interpret backend supports only operations which have reference implementations. The scope of supported operation is extending (it is needed for tests) but coverage is not 100%.

BBTW why template plugin should support to infer all operations form the last opset?

Reply:
Why not?